README.md

```diff
@@ -247,15 +246,15 @@ CTest will only display the standard error output of tests that failed. In order
 
 ## Running A Subset Of The Tests
 
-`${PROJECT_BINARY_DIR}\tests\utils\stl-lit\stl-lit.py` can be invoked on a subdirectory of a testsuite and will execute
-all the tests under that subdirectory. This can mean executing the entirety of a single testsuite, running all tests
+`${PROJECT_BINARY_DIR}\tests\utils\stl-lit\stl-lit.py` can be invoked on a subdirectory of a test suite and will execute
+all the tests under that subdirectory. This can mean executing the entirety of a single test suite, running all tests

```

Comment:
Adding a space to "test suite" seems reasonable (it doesn't seem to be used as a single word very often, there's no strong argument for doing so unlike "spacetime", and this aligns with [LLVM lit's documentation](https://llvm.org/docs/CommandGuide/lit.html) which almost exclusively uses a space), but there are more occurrences that should be updated.